ci/eval/compare/maintainers.nix

```
@@ -105,9 +105,9 @@ let
   ) attrsWithModifiedFiles;
 
   byMaintainer = lib.groupBy (ping: toString ping.id) listToPing;
-
-  packagesPerMaintainer = lib.mapAttrs (
-    maintainer: packages: map (pkg: pkg.packageName) packages
-  ) byMaintainer;
 in
-packagesPerMaintainer
+{
+  maintainers = lib.mapAttrs (_: pkgs: map (pkg: pkg.packageName) pkgs) byMaintainer;
```

Could be simplified:


```suggestion
  maintainers = lib.mapAttrs (_: map (pkg: pkg.packageName)) byMaintainer;
```

Or if we don't need to be strict about `pkg ? packageName` on all items, we can use `builtins.catAttrs`:


```suggestion
  maintainers = lib.mapAttrs (_: lib.catAttrs "packageName") byMaintainer;
```